wit/deps/spin@unversioned/world.wit

```diff
@@ -23,7 +23,7 @@ world platform {
   import postgres;
   import mysql;
   import sqlite;
-  import redis;
+  import redis;  
```

Comment:
```suggestion
  import redis;
```

(accidental whitespace change)

Reply:
ehh... let me fix that, wish linting could check that as well.